pkg/apis/batch/types.go

```diff
@@ -223,6 +223,10 @@ type PodFailurePolicy struct {
 	// the backoffLimit. At most 20 elements are allowed.
 	// +listType=atomic
 	Rules []PodFailurePolicyRule
+	// Only allow pods to be started once the terminating pods have been
+	// fully deleted.
+	// +optional
+	DelayPodRecreationUntilTerminal *bool
```

Comment:
Yet thinking, given the current implementation should we maybe consider another name, say: 
`TerminatingAsActive`? Such a name would be better to express that the count of `Active` is also affected by this change.